dev/tests/integration/testsuite/Magento/Cms/Model/BlockTest.php

```diff
@@ -0,0 +1,49 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+namespace Magento\Cms\Model;
+
+use Magento\Backend\Model\Auth\Session;
+use Magento\Cms\Api\BlockRepositoryInterface;
+use Magento\Framework\Stdlib\DateTime\DateTime;
+use Magento\TestFramework\Helper\Bootstrap;
+use Magento\User\Model\User;
+use PHPUnit\Framework\TestCase;
+
+/**
+ * @magentoAppArea adminhtml
+ */
+class BlockTest extends TestCase
+{
+    protected function setUp()
+    {
+        $user = Bootstrap::getObjectManager()->create(
+            User::class
+        )->loadByUsername(
+            \Magento\TestFramework\Bootstrap::ADMIN_NAME
+        );
+
+        /** @var $session Session */
+        $session = Bootstrap::getObjectManager()->get(
+            Session::class
+        );
+        $session->setUser($user);
+    }
+
+    /**
+     * @magentoDbIsolation enabled
+     */
+    public function testUpdateTime()
+    {
+        $objectManager = Bootstrap::getObjectManager();
```

Comment:
It's better to get the object manager in the `setUp` method ([example](https://github.com/magento/magento2/blob/3e3e8fc90ace43ef7113e52f2c1e7bc20c78d94a/dev/tests/integration/testsuite/Magento/Catalog/Model/Product/CopierTest.php#L67)).

Reply:
Added ObjectManager to Setup.

I copied test from `dev/tests/integration/testsuite/Magento/Cms/Model/PageTest.php`